### ISSUE TITLE
Uncaught exception on startup (macOS 10.11, GitUp release v1.2)

### ISSUE THREAD

**Issue by elder-n00b** [git-up/GitUp#752]
Mac OS X 10.11, GitUp v1.2 installed by "brew install homebrew/cask/gitup" (should be the same v1.2 release available here). More details in the attached crash reports. More than actually useful I'd say.
I'm just trying it out and I get "internal error" at startup because NSInvalidArgumentException on [NSApplication setAppearance:]. If I choose "continue" it appears to work without problem. Though I wonder how safe is it to do so. Also when the OS finally asks "The last time you opened GitUp, it unexpectedly quit while reopening windows. Do you want to try to reopen its windows again?" and I click "Don't reopen" it proceeds to work without the exception nor any apparent problem. The next time, "internal error" again. Yet it works. 
Too bad my obsolete Xcode 7.3.1 ~(the best I can get on 10.11 apparently?)~ refuses to open the project file (any way to build from command line?). I thought I could try with with the brew source formula, but it seems to be another unrelated project (https://github.com/earwig/git-repo-updater, "stable 0.5.1, HEAD", python code). I tried tricking Xcode changing all the "minToolsVersion" and "minSystemVersion" to 7.0 without success. I have near zero experience with macOS, Xcode, and ObjC though, I bet I missed something (and I apologize if all this turns out to be my fault). 
My best guess so far, this is almost a non issue after all, probably a minor incompatibility with older OS version, and the fix (assuming there's an interest to support 10.11, I hope so) would be to catch the exception or better yet check the OS version first. 
[gitup__random-startup-crash_1.txt](https://github.com/git-up/GitUp/files/6245453/gitup__random-startup-crash_1.txt)
[gitup__random-startup-crash_2.txt](https://github.com/git-up/GitUp/files/6245454/gitup__random-startup-crash_2.txt)
ps: file names say "random" and "crash", my mistake.

**Comment by lolgear** [comment]
```
Crashing on exception: -[NSApplication setAppearance:]: unrecognized selector sent to instance
```

**Comment by lucasderraugh** [comment]
Actually pretty easy to address this issue for backwards compatibility. For some reason the API doesn't warn about using it pre 10.14 but it would appear that it's not available before that release.
```objc
@property (nullable, strong) NSAppearance *appearance API_AVAILABLE(macos(10.14));
```

**Comment by lolgear** [comment]
Is it possible to increase a deployment target number to 10.11?

**Comment by lucasderraugh** [comment]
> Is it possible to increase a deployment target number to 10.11?

Yes, but it's a bit off topic for this discussion. I think I'll make a formal document for when we bump versions. It's a bit confusing what's the currently supported OS.

**Comment by lucasderraugh** [comment]
Fixed in 1.3 (currently in continuous channel)
